packages/docs/scripts/pages.json

```diff
@@ -70,6 +70,11 @@
     "href": "https://sanyamjainqwik.vercel.app/",
     "tags": "portfolio"
   },
+  {
+    "href": "https://www.blackflag.dev/",
+    "size": "large",
```

Comment:
please reduce this to 2 props - href and size
thanks!

Reply:
imo it should be `href` and `tag`, right?
@openblackflag do you have a minute to adjust that so we can merge the PR?